.travis.yml

```diff
@@ -31,10 +31,32 @@ jobs:
       services: [docker]
       dist: xenial
       python: "3.7"
+      env:
+        - IMAGE_NAME=praekeltfoundation/malariaconnect
```

Comment:
You'll need to have created the repo for this image on Docker Hub and given the `automation` team write access. (If you haven't already).